src/ScioSense_ENS160.h

```diff
@@ -98,6 +98,7 @@
 class ScioSense_ENS160 {
 		
 	public:
+	    ScioSense_ENS160(TwoWire *i2c, u_int8_t slaveaddr = ENS160_I2CADDR_0);               				// Constructor using slave address (5A or 5B)

```

Comment:
Why `u_int8_t` here? Should that be `uint8_t`?

Reply:
Good spot! Fixed.